src/steps/ms-defender/device/converters.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  createIntegrationEntity,
+  createDirectRelationship,
+  Entity,
+  Relationship,
+  RelationshipClass,
+  createMappedRelationship,
+} from '@jupiterone/integration-sdk-core';
+import { Machine } from '../../../types';
+import { entities, MappedRelationships, TargetEntities } from '../constants';
+
+export function createAccountMachineRelationship(
+  account: Entity,
+  machine: Entity,
+): Relationship {
+  return createDirectRelationship({
+    _class: RelationshipClass.HAS,
+    from: account,
+    to: machine,
+  });
+}
+export function createMachineEntity(data: Machine): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: {}, // removed due to size
```

Comment:
Is this the case in fact? Was a test performed and it was found that the machine data is too large to include? We should include the source data.